components/List/tests/unit/MDCListThemerTest.m

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialList+ColorThemer.h"
+#import "MaterialList+ListThemer.h"
+#import "MaterialList+TypographyThemer.h"
+#import "MaterialList.h"
+
+#import <XCTest/XCTest.h>
+
+#ifndef CGFLOAT_EPSILON
```

Comment:
_Tip:_ I think you can get rid of this an `#import MaterialMath.h` and use the `MDCCGFloatEqual` function